contrib/terraform/upcloud/modules/kubernetes-cluster/main.tf

```diff
@@ -265,6 +355,96 @@ resource "upcloud_firewall_rules" "k8s" {
       source_address_start   = "0.0.0.0"
     }
   }
+
+  dynamic firewall_rule {
+    for_each = var.worker_allowed_ports
+
+    content {
+      action                 = "accept"
+      comment                = "Allow access on this port"
+      destination_port_end   = firewall_rule.value.port_range_max
+      destination_port_start = firewall_rule.value.port_range_min
+      direction              = "in"
+      family                 = "IPv4"
+      protocol               = firewall_rule.value.protocol
+      source_address_end     = "255.255.255.255"
```

Comment:
Can you add that you can specify ip range? This would also allow you to remove ssh allow/drop.

Reply:
I can add that. I'm not sure about removing the separate ssh allow/drop variable, I feel like it makes sense to keep it (even if it's technically redundant) since it's one of the ports that pretty much everyone will want to allow in. Same with the master API port.